### DISCUSSION TITLE
censys form in yaml file

### DISCUSSION THREAD

**Discussion by molitona** [projectdiscovery/subfinder#633] [Q&A] [answered]
Hi, should censys be added like that in yaml file ?

```

censysUsername: []
- 1
- 2
censysPassword: []
- 3
- 4
```
and if i've 2 accounts will 1 and 3 used together / 2 and 4 used together ?

because there is API ID and Secret in censys after auth.

**Comment by ehsandeep** [comment, marked as answer]
@molitona, here is a valid example:

```yaml
censys:
  - api_id_1:secret_1
  - api_id_2:secret_2
  - api_id_3:secret_3

```
